cpp/data/NewsletterUploadForm.html

```diff
@@ -0,0 +1,11 @@
+<html>
+  <head>
+    <title>Newsletter Upload Form</title>
+  </head>
+  <body>
+    <form enctype="multipart/form-data" action="/var/www/cgi-bin/newsletter_uploader" method="post">
+      <p>File: <input type="file" name="file"></p>
```

Comment:
What exactly is the structure of the file? Please provide documentation.

Reply:
Documented in the usage message of the newsletter_uploader program and in NewsletterUploadForm.html.